substrate/frame/revive/src/limits.rs

```diff
@@ -48,6 +48,9 @@ pub const CALL_STACK_DEPTH: u32 = 25;
 /// We set it to the same limit that ethereum has. It is unlikely to change.
 pub const NUM_EVENT_TOPICS: u32 = 4;
 
+/// Maximum size of events (including topics) and storage values.
```

Comment:
isn't that a dup from (down below)

/// Maximum payload size of events.
pub const EVENT_BYTES: u32 = 64 * 1024;

Reply:
Yes, something here got messed up during a rebase I did early on. Somehow I wrongly reintroduced `PAYLOAD_BYTES`. Fixed it in https://github.com/paritytech/polkadot-sdk/pull/10166/commits/116ba4f6d7596a70c912585d02f44647a57f5d66.